sway/ipc-json.c

```diff
@@ -23,6 +23,20 @@
 static const int i3_output_id = INT32_MAX;
 static const int i3_scratch_id = INT32_MAX - 1;
 
+static const char *ipc_json_node_type_description(enum sway_node_type l) {
```

Comment:
Nit: let's use something more descriptive than `l` here (`l` is used for `sway_container_layout` elsewhere in this file, which is bad as-is; making it represent a `sway_node_type` here makes things more confusing).

Reply:
Renamed to `node_type`.